packages/api/tests/lib/reconciliationReport/test-normalizeEvent.js

```diff
@@ -48,30 +36,13 @@ test('normalizeEvent moves input key collectionId array to array on collectionId
   const actual = normalizeEvent(inputEvent);
   t.deepEqual(actual, expect);
 });
-test('normalizeEvent adds new collectionIds key when collectionId passed to Internal report', (t) => {
-  const inputEvent = {
-    systemBucket: 'systemBucket',
-    stackName: 'stackName',
-    startTimestamp: new Date().toISOString(),
-    endTimestamp: new Date().toISOString(),
-    reportType: 'Internal',
-    collectionId: 'someCollection___version',
-  };
-  const expect = {
-    ...inputEvent,
-    collectionIds: ['someCollection___version'],
-  };
-
-  const actual = normalizeEvent(inputEvent);
-  t.deepEqual(actual, expect);
-});
```

Comment:
Shouldn't this still be a valid test?

Reply:
Or rather the expect should have collectionId removed from inputEvent. but I still think this is valid if you do that in the expect. like     ...omit(inputEvent, ['collectionId'])

But maybe it's covered below in another test?

Reply:
it's covered by other tests with `reportType: 'anytype'`